### PR TITLE
adds a toolbar

### PR DESCRIPTION
**Pull request by collin**
Open in a web browser and see that there is a toolbar.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# THIS HAS A TOOLBAR
```
Comment (collin) [inline README.md:1]: Hey this looks great!

File: client/app.js [hunk 2]
```diff
@@ -3,6 +3,8 @@ import React from 'react'
 import {Navbar} from './components'
 import Routes from './routes'
 
+console.log('this should be here')
```
Comment (collin) [inline client/app.js:6]: Hey, this is also great, but let's add an "!"!

**Review by collin — COMMENTED** [review]
Haha, my own PR! It looks great.

**Comment by benwilhelm** [comment]
Duplicate of #4

**Pull request closed without merging**